src/forms.css

```diff
@@ -72,27 +68,36 @@
  * Style a text input. Set the color of the border with a `input--border-{color}` modifier (`*-dark` colors are not available).
  *
  * @memberof Inputs & textareas
+ * @selectors .input
  * @example
  * <input class='input' placeholder='default' />
  * <input class='input input--border-green w240' placeholder='green' />
  */
-.input {
-  height: 36px;
-  line-height: 34px; /* minus border */
-  padding: 0 12px;
-}
 
 /**
  * Make an input small.
  *
  * @memberof Inputs & textareas
  * @example
- * <input class='input input--s txt-s' placeholder='small' />
+ * <input class='input input--s' placeholder='small' />
  */
 .input--s {
-  height: 24px;
-  line-height: 22px; /* minus border */
-  padding: 0 6px;
+  font-size: var(--font-size-s);
+  line-height: var(--line-height-s);
+  padding: 3px 6px;
+}
+
+/**
+ * Make an input extra small.
+ *
+ * @memberof Inputs & textareas
+ * @example
+ * <input class='input input--xs' placeholder='extra small' />
+ */
+.input--xs {
```

Comment:
This is ridiculously small. We want this?

Reply:
We use similar inputs in studio by add txt-xs and padding rules to `.input`. Would be cleaner to have access to this!

Reply:
Ok. Yipes.